src/librbd/CMakeLists.txt

```diff
@@ -24,11 +24,13 @@ set(librbd_internal_srcs
   LibrbdWriteback.cc
   MirroringWatcher.cc 
   ObjectMap.cc
-  ObjectWatcher.cc 
   Operations.cc
   Utils.cc
   cache/ImageWriteback.cc
   cache/PassthroughImageCache.cc
+  watcher/Watcher.cc
+  watcher/WatcherTypes.cc
```

Comment:
Minor: can be renamed to "watcher/Types.cc" (and associated header file) since the second "Watcher" is redundant.